sqlitecluster/SQLiteNode.cpp

```diff
@@ -2034,7 +2030,7 @@ void SQLiteNode::_queueSynchronize(Peer* peer, SData& response, bool sendAll) {
     _queueSynchronizeStateless(peer->nameValueMap, name, peer->name, _state, (unsentTransactions.load() ? _lastSentTransactionID : _db.getCommitCount()), _db, response, sendAll);
 }
 
-void SQLiteNode::_queueSynchronizeStateless(const STable& params, const string& name, const string& peerName, int _state, uint64_t targetCommit, SQLite& db, SData& response, bool sendAll) {
+void SQLiteNode::_queueSynchronizeStateless(const STable& params, const string& name, const string& peerName, State _state, uint64_t targetCommit, SQLite& db, SData& response, bool sendAll) {
```

Comment:
am i missing something? `_state` seems to be unused in this function all together.

Reply:
I thought the same thing and so I removed it but it turns out it's used inside the `PXXXX` log macros, because of this line:
```
#define SLOGPREFIX "{" << name << "/" << SQLiteNode::stateName(_state) << "} "
```

Reply:
🤦‍♂